TicTacToeApp/Board.swift

```diff
@@ -0,0 +1,48 @@
+
+public class Board {
+  
+  private let markedSpaces: [Int: PlayerMark]
+  private let size = 3
+
+  public init() {
+    markedSpaces = [:]
+  }
+
+  public init(markedSpaces: [Int : PlayerMark]) {
+    self.markedSpaces = markedSpaces
+  }
+
+  public func markSpace(id: Int, mark: PlayerMark) -> Board {
```

Comment:
In this context, having `id` as part of the name is almost akin to Hungarian notation. I would consider renaming it to just `space` or even `position`.